EPlast/EPlast/gulpfile.js

```diff
@@ -28,6 +29,12 @@ gulp.task('less', function () {
         .pipe(gulp.dest(paths.webroot + 'css/lessCompiled'));
 });
 
+gulp.task("sass", function () {
+    return gulp.src('wwwroot/less/**/*.scss')
```

Comment:
you should have a separate folder for scss scripts, or the name of the folder should be generic to both less and scss